library-best-practices/JLBP-19.md

```diff
@@ -0,0 +1,23 @@
+[JLBP-19] Place each package in a single jar
+--------------------------------------------------
+
+Every package should have a single Maven group ID and artifact ID.
```

Comment:
I really want to stay away form the word "overlap" here. It is not commonly used, and we've already defined it to mean the same class in multiple artifacts.

It's also tricky because it is OK for multiple Maven packages to contain the same packages as long as they have the same group ID and artifact ID, but different versions. That is, it's not Maven artifacts at issue here.

Anyone see a better way to phrase this? Maybe we should start by describing the problem purely in the context of jars and then expand that to cover maven coordinates?

Reply:
Maybe just change the verb: "Every package should be contained in only one Maven group ID and artifact ID."

Reply:
Does "Maven artifact" in documentation really refer to the version as well? Wouldn't you say "foo:bar:1" and "foo:bar:2" are two versions of the same artifact, not two artifacts with the same group ID and artifact ID?